database/schema.sql

```diff
@@ -0,0 +1,87 @@
+DROP DATABASE IF EXISTS product_overview;
+CREATE DATABASE product_overview;
+\c product_overview;
+
+DROP TABLE IF EXISTS product_list CASCADE;
+
+CREATE TABLE product_list (
+  id  SERIAL NOT NULL PRIMARY KEY,
+  name TEXT NOT NULL,
+  slogan TEXT NOT NULL,
+  description TEXT NOT NULL,
+  category TEXT NOT NULL,
+  default_price TEXT NOT NULL
+);
+
+CREATE TABLE features (
+  id SERIAL NOT NULL PRIMARY KEY,
+  product_id INT NOT NULL,
+  feature TEXT NOT NULL,
+  value TEXT NOT NULL
+);
+
+CREATE TABLE related (
+  id SERIAL NOT NULL PRIMARY KEY,
+  current_product_id INT NOT NULL,
+  related_product_id INT NOT NULL
+);
+
+CREATE TABLE styles (
+  id SERIAL NOT NULL PRIMARY KEY,
+  product_id INT NOT NULL,
+  name TEXT NOT NULL,
+  sale_price TEXT,
+  original_price TEXT NOT NULL,
+  default_style BOOLEAN NOT NULL DEFAULT false
+);
+
+CREATE TABLE photos (
+  id SERIAL NOT NULL PRIMARY KEY,
+  styleId INT NOT NULL,
```

Comment:
I just wanted to point out the column name here. I think you intended it to be `style_id`...

Reply:
then again you might not have 😅